data/Course Details/LKM400 - BAHASA MALAYSIA.txt

```diff
@@ -1,3 +1,3 @@
-Tuesday 5:00pm-7:00pm
+Tuesday 5:00pm-7:00pm ok
```

Comment:
This one I am trying the code and I forget to remove it. I will remove it when I push again.

Reply:
Did you remove? @GitHubUser5136

Reply:
I removed in my side. Now the "ok" is not there any more but I not sure why here still got